internal/printers/table.go

```diff
@@ -4,23 +4,31 @@ import (
 	"io"
 
 	"github.com/olekukonko/tablewriter"
+	"github.com/olekukonko/tablewriter/renderer"
+	"github.com/olekukonko/tablewriter/tw"
 )
 
 // PrintTable writes an terminal-friendly table of the values to the target.
 func PrintTable(target io.Writer, headers []string, rows [][]string) {
-	table := tablewriter.NewWriter(target)
-	table.SetHeader(headers)
-	table.SetAutoWrapText(false)
-	table.SetAutoFormatHeaders(true)
-	table.SetHeaderAlignment(tablewriter.ALIGN_LEFT)
-	table.SetAlignment(tablewriter.ALIGN_LEFT)
-	table.SetCenterSeparator("")
-	table.SetColumnSeparator("")
-	table.SetRowSeparator("")
-	table.SetHeaderLine(false)
-	table.SetBorder(false)
-	table.SetTablePadding("\t")
-	table.SetNoWhiteSpace(true)
-	table.AppendBulk(rows)
-	table.Render()
+	table := tablewriter.NewTable(target,
+		tablewriter.WithRenderer(renderer.NewBlueprint()),
+		tablewriter.WithRowAutoWrap(tw.WrapNone),
+		tablewriter.WithHeaderAutoFormat(tw.On),
+		tablewriter.WithHeaderAlignment(tw.AlignLeft),
+		tablewriter.WithRowAlignment(tw.AlignLeft),
+		tablewriter.WithRendition(tw.Rendition{
+			Symbols: tw.NewSymbolCustom("custom").WithCenter("").WithColumn("").WithRow(""),
+			Settings: tw.Settings{
+				Lines: tw.LinesNone,
+				Separators: tw.Separators{
+					BetweenColumns: tw.On,
+				},
+			},
+			Borders: tw.BorderNone,
+		}),
+		tablewriter.WithTrimSpace(tw.Off),
+		)
```

Comment:
I made these updates based on [the migration guide](https://github.com/olekukonko/tablewriter/blob/master/MIGRATION.md) and then validated them locally. This is only used for `zed context list` output and I verified that it looks the same as on current main.